### PR TITLE
NO-ISSUE: upgrade google storage

### PR DESCRIPTION
**Pull request by adriengentil**
- Bump google-cloud-storage from 2.5.0 to 2.6.0
- Cleanup temporary build directory that prevents the tests to run in Prow

### CODE REVIEW

**Comment by osherdp** [comment]
/hold
I believe those changes are meant for debugging. So holding to make it safe

**Review by osherdp — APPROVED** [review]

**Comment by adriengentil** [comment]
Finally! :tada:

File: Makefile
Comment (osherdp) [inline Makefile:24]: *in order

File: pyproject.toml
Comment (osherdp) [inline pyproject.toml:3]: The pinning is required for the fix? Or just for good measures?

File: Makefile
Comment (osherdp) [inline Makefile:11]: An improvement for the next PR is to do every job via tox, as it should operate in separate work dir

**Review by osherdp — CHANGES_REQUESTED** [review]

File: Makefile
Comment (adriengentil) [reply Makefile:11]: yeah, we should review the build scripts

File: pyproject.toml
Comment (adriengentil) [reply pyproject.toml:3]: yes, it's for sanity, and I expect renovate to update them :crossed_fingers:

**Review by osherdp — APPROVED** [review]

**Pull request merged**
